fhir-server-test/src/test/nodejs/nats-subscriber.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+"use-strict";
+
+var server = 'nats://127.0.0.1:4222'
+var stan = require('node-nats-streaming').connect('nats-streaming', 'nats-subscriber', server);
+var subscription
+
+stan.on('connect', function () {
+  console.log('Waiting for incoming messages...Ctrl-C to exit');
+  let opts = stan.subscriptionOptions().setStartWithLastReceived()
+  subscription = stan.subscribe('FHIRNotificationEvent', opts)
+  subscription.on('message', function (msg) {
+    console.log('Received a message [' + msg.getSequence() + '] ' + msg.getSubject() + ' ' + msg.getData());
+  })
+})
+
+stan.on('close', function() {
+  process.exit()
+})
+
+function cleanup() {
+  subscription.unsubscribe();
+  subscription.on('unsubscribed', function() {
+    console.log('\nUnsubscribed and closing connection');
+    stan.close();
+  })
+}
+
```

Comment:
```suggestion
stan.on('connect', function () {
  console.log('Waiting for incoming messages...Ctrl-C to exit')
  let opts = stan.subscriptionOptions().setStartWithLastReceived()
  subscription = stan.subscribe('FHIRNotificationEvent', opts)
  subscription.on('message', function (msg) {
    console.log('Received a message [' + msg.getSequence() + '] ' + msg.getSubject() + ' ' + msg.getData())
  })
})

stan.on('close', function() {
  process.exit()
})

function cleanup() {
  subscription.unsubscribe()
  subscription.on('unsubscribed', function() {
    console.log('\nUnsubscribed and closing connection')
    stan.close()
  })
}

```
consistency on the line endings

Reply:
we don't have any code style recommendations for javascript...this is our first one :-)
that said, I agree it should be internally consistent if possible.

personally, i prefer to always have the `;` at the end...mostly because i don't care to think about the cases where its needed vs not needed

Reply:
as do I, I think it should be consistent.

Reply:
I learned to give up the ';' on my last project, but it was tough.  Looks like a couple snuck in here anyway, but my goal was to not use any.

Reply:
I ended up adding the ';' in everywhere to address paul's comment.  I'd also be fine with removing them all, but I didn't see your comment until I had already added them.  :-)

Reply:
That's fine and probably what most will be comfortable with.